cheatsheets/Forgot_Password_Cheat_Sheet.md

```diff
@@ -1,60 +1,104 @@
-# Introduction
+# Forgot Password and MFA Cheat Sheet
```

Comment:
Since it is the title, shouldn't be Multi-factor Authentication instead of MFA?

Reply:
I'm still considering the focus properly.